src/recommendations/RecommendationsList.jsx

```diff
@@ -41,10 +44,12 @@ RecommendationsList.propTypes = {
     })),
     marketingUrl: PropTypes.string.isRequired,
   })),
+  userId: PropTypes.number,
 };
 
 RecommendationsList.defaultProps = {
   recommendations: [],
+  userId: null,
```

Comment:
should default value not be number ?

Reply:
No not a number because number can be the actual user id.